toqito/matrices/fourier.py

```diff
@@ -48,13 +46,19 @@ def fourier(dim: int) -> np.ndarray:
     .. bibliography::
         :filter: docname in docnames
 
+    Parameters
+    ----------
+    dim : int
+        The size of the Fourier matrix.
```

Comment:
So I think we do want to maintain the convention for specifying parameters as we had before:

```suggestion
    :param dim: The size of the Fourier matrix.
    :return: The Fourier matrix of dimension :code:`dim`.
```

Part of the reason for this is consistency, but the other part is that conveying the type of `dim` is redundant and already provided via the arguments in the function prototype.